File: 350-intersection-of-two-arrays-ii/350-intersection-of-two-arrays-ii.cpp
class Solution {
public:
    vector<int> intersect(vector<int>& nums1, vector<int>& nums2) {
        vector<int>answer;
        int n=nums2.size();
        vector<int>check(n,1);           
        for(int i=0;i<nums1.size();i++)
            for(int j=0;j<nums2.size();j++)
                if(nums1[i]==nums2[j]&&check[j])
                   {
                    answer.push_back(nums1[i]);
                    check[j]=0;
                    break;
                    
                   }
        return answer;
    }
};

File: 350-intersection-of-two-arrays-ii/350-intersection-of-two-arrays-ii.py
class Solution:
    def intersect(self, nums1: List[int], nums2: List[int]) -> List[int]:
        ans=[]
        for i in nums1:
            for j in nums2:
                if i==j:
                    ans.append(i)
                    nums2.remove(j)
                    break
        return ans
                    
        